File: ce/customerengagement/on-premises/snippets/csharp/CRMV8/discoveryservice/cs/discoveryservice3.cs
// <snippetdiscoveryservice3>


 // Retrieve details about a single organization discoverable via the Discovery service.
 //
 RetrieveOrganizationRequest orgRequest =
     new RetrieveOrganizationRequest()
     {
         UniqueName = organizations.Details[organizations.Details.Count -1].UniqueName,
         AccessType = EndpointAccessType.Default,
         Release = OrganizationRelease.Current
     };
 RetrieveOrganizationResponse org =
     (RetrieveOrganizationResponse)service.Execute(orgRequest);

// </snippetdiscoveryservice3>

File: ce/customerengagement/on-premises/snippets/visualbasic/CRMV8/discoveryservicevb/vb/discoveryservice3.vb
' <snippetdiscoveryservice3>


 ' Retrieve details about a single organization discoverable via the
 ' Discovery service.
 Dim orgRequest As New RetrieveOrganizationRequest() With { _
     .UniqueName = organizations.Details(organizations.Details.Count - 1).UniqueName, _
     .AccessType = EndpointAccessType.Default, .Release = OrganizationRelease.Current}
 Dim org As RetrieveOrganizationResponse = CType(service.Execute(orgRequest), RetrieveOrganizationResponse)

' </snippetdiscoveryservice3>